templates/cinder/config/cinder.conf

```diff
@@ -65,19 +72,11 @@ user_domain_name = Default
 project_name = service
 project_domain_name = Default
 
-#TODO
-#[service_user]
-#send_service_user_token = True
-#TODO
-#auth_url = http://keystone.openstack.svc:5000/
-#project_name = service
-#project_domain_name = Default
-#username = cinder
-#user_domain_name = Default
-#auth_type = password
-
-[oslo_concurrency]
-lock_path = /var/locks/openstack/cinder
-
-[os_brick]
-lock_path = /var/locks/openstack/os-brick
+[service_user]
```

Comment:
nit: it would have been good to mention that we are setting the password in `init.sh`.

Reply:
Then we need to find a way to document this for developers.